tests/test_commands.py

```
@@ -316,3 +316,35 @@ def test_add_sage_feature_help():
+            "CID",
+            "--feature_generators",
+            "ms2pip,deeplc",
+            "--id_decoy_pattern",
+            "^DECOY_",
+            "--output",
+            "{}/UPS1_50amol_R1_rescored.idXML".format(local_folder),
+            "--ms2_tolerance",
+            "0.05",
+        ],
+    )
+    assert 0 == result.exit_code
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `result.exit_code == 0`

(SIM300)